athom-smart-plug-v2.yaml

```diff
@@ -84,27 +84,31 @@ sensor:
     update_interval: 60s
 
   - platform: cse7766
-    update_interval: 10s
     current:
       name: "Current"
       filters:
-          - lambda: if (x < 0.060) return 0.0; else return x;   #For the chip will report less than 3w power when no load is connected
+        - throttle_average: 10s
+        - lambda: if (x < 0.060) return 0.0; else return x;   #For the chip will report less than 3w power when no load is connected
 
 
     voltage:
       name: "Voltage"
+      filters:
+        - throttle_average: 10s
     power:
       name: "Power"
       id: power_sensor
       filters:
-          - lambda: if (x < 3.0) return 0.0; else return x;    #For the chip will report less than 3w power when no load is connected
+        - throttle_average: 10s
```

Comment:
@hellotomtom Not sure if you noticed this but this will break the Total Energy sensor as it will show `nan` as the data back to home assistant. 

I found that setting this to `- delta: 0.001` fixes that issue. I suspect that the throttle average is doing something to this value that breaks Total Energy.